### ISSUE TITLE
Strange Behavior with block-closing-brace-space-after

### ISSUE THREAD

**Issue by thegranddesign** [kristerkari/stylelint-scss#275]
### Minimal Config

```stylelintrc
{
  "rules": {
    "block-closing-brace-space-after": "never"
  }
}
```

### Stylesheet to Reproduce

```scss
@font-face {
}

@font-face {
}
```

I get `block-closing-brace-space-after: Unexpected whitespace after "}"` even though the config is set to `never`.  I'm assuming it's matching the newline which it should not since that is handled by `block-closing-brace-newline-after`.  There's literally no way to make this pass except to only have one block in the file.
